Makefile

```diff
@@ -17,7 +17,7 @@ test-flake8: virtualenv
 	${VIRTUALENV_ROOT}/bin/flake8 .
 
 .PHONY: test-python
-test-python: virtualenv
+test-python: virtualenv requirements-dev
```

Comment:
I'm personally not a fan of having `make test` install requirements, because I sometimes want to run `make test` a lot quickly and having it run pip every time is annoying... on the other hand, there's definitely an advantage to having make do all the stuff magically for you.

I'm guessing I'm saying that maybe we should talk as a team about what we want our Makefiles to be like? One things for sure, they are in need of a rewrite (they could be a lot more consistent, and probably simpler too.)

Reply:
The majority of our apps (and repos with Python tests) include the dev-requirements in the `make test` command, so this will bring it in line with those.

Agree that the Makefiles need a review!

Reply:
> The majority of our apps (and repos with Python tests) include the dev-requirements in the make test command, so this will bring it in line with those.

That's fair, this change makes sense then.

> Agree that the Makefiles need a review!

It's on my personal firebreak list 😆